contrib/mesos/pkg/scheduler/constraint/doc.go

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package constraint exposes Marathon-like constraints for scheduling pods.
+// Incomplete.
```

Comment:
I realize this says "Incomplete" but it wasn't clear to me where you are going with it. Am I correct in understanding that this can only ever possibly work if Kubernetes eventually supports scheduling constraints that are compatible with the Mesos ones? (That is, at a minimum, we would need the Kubernetes API to allow to specify in Pod definitions scheduling constraints with similar semantics to Mesos's.)